pom.xml

```diff
@@ -260,6 +261,7 @@ under the License.
               <goals>
                 <goal>site</goal>
               </goals>
+              <streamLogsOnFailures>true</streamLogsOnFailures>
```

Comment:
What does this change in this case?

Reply:
because when running IT locally it's more convenient to have the output of failed test in the console rather than looking into a build.log file.

Reply:
yes because it helps me when writing/debugging IT test.